isbnbot/normalize_isbns.py

```diff
@@ -0,0 +1,158 @@
+"""
+normalize ISBNs
+NOTE: This script assumes the Open Library Dump passed only contains editions with an isbn_10 or isbn_13
+"""
+import argparse
+import datetime
+import isbnlib
+import gzip
+import json
+import logging
+import sys
+
+from olclient.openlibrary import OpenLibrary
+from os import makedirs
+
+
+ALLOWED_ISBN_CHARS = {'0', '1', '2','3', '4', '5', '6', '7', '8', '9', 'X', '-'}
```

Comment:
:) you could make this a bit easier on yourself by doing:

`set(str(x) for x in range(0, 10) + ['X', '-'])`

Not sure we want `-` in this list.

Reply:
I think it's good as is easier to read with the characters explicit; the `-` are needed because those are the ISBNs we want to normalize! :P